app/mm-redux/selectors/entities/users.ts

```diff
@@ -480,6 +480,27 @@ export function makeGetProfilesByIdsAndUsernames(): (a: GlobalState, b: Array<$I
     );
 }
 
+export function makeGetProfilesByIds(): (a: GlobalState, b: Array<$ID<UserProfile>>) => Array<UserProfile> {
+    return createSelector(
+        getUsers,
+        (state: GlobalState, allUserIds: Array<$ID<UserProfile>>) => allUserIds,
```

Comment:
a bit confusing as I thought it was ALL user profiles, perhaps change it to `userIds`